src/dcrypto/v1/v1_test.go

```diff
@@ -75,7 +77,7 @@ func TestRoundTrip(t *testing.T) {
 				t.Errorf("ioutil.ReadAll(*EncryptReader) => %q; want nil", err)
 				continue
 			}
-			dr, err := newDecryptReader(bytes.NewBuffer(cipher), password, 1024)
+			dr, err := NewDecryptReader(bytes.NewBuffer(cipher), password)
```

Comment:
I just noticed that we could do `bytes.NewReader(cipher)` since we don't need to write to the buffer that is passed into NewDecryptReader as io.Reader.